test/rules/src/https_everywhere_checker/check_rules.py

```diff
@@ -99,13 +99,14 @@ def processTask(self, task):
         problems = []
         for url in task.urls:
             result = self.processUrl(url, task)
-            if result:
+            if result[0]:
                 problems.append(result)
         if problems:
             for problem in problems:
-                logging.error("{}: {}".format(task.ruleFname, problem))
+                logging.error("%s: %s" % (task.ruleFname, problem[0]))
```

Comment:
Let's preference the curly-braces string formatting: https://realpython.com/python-f-strings/